src/navigation/stacks/main/index.tsx

```diff
@@ -2,16 +2,15 @@ import React from 'react';
 
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 
-import { StackParamList } from 'navigation/types';
-import { AppScreens } from 'navigation/types';
```

Comment:
why stop using the constant for the screen names?

Reply:
we are using the advantage of types in case theres any change the linter will warn us